core-web/libs/edit-content/src/lib/models/dot-edit-content-host-folder-field.interface.ts

```diff
@@ -0,0 +1,20 @@
+import { TreeNode } from 'primeng/api';
```

Comment:
If this only belongs to the new field can be inside the `dot-edit-content-host-folder-field`

Reply:
@oidacra But these types are reused for DotEditContentService, I think is a good place to just import the types.